build_all/windows/build.cmd

```diff
@@ -68,10 +66,6 @@ if %build-platform% == x64 (
 )
 goto args-continue
 
-:arg-use-websockets
-set CMAKE_use_wsio=ON
-goto args-continue
-
 :arg-build-nuget
 shift
 if "%1" equ "" call :usage && exit /b 1
```

Comment:
Interesting how I missed this. Removed.

---
In reply to: [107512198](https://github.com/Azure/azure-uamqp-c/pull/58#discussion_r107512198) [](ancestors = 107512198)